src/core/client/admin/routes/Configure/Header.tsx

```diff
@@ -1,21 +1,27 @@
 import cn from "classnames";
-import React, { FunctionComponent } from "react";
-
-import { Typography } from "coral-ui/components";
-import { PropTypesOf } from "coral-ui/types";
+import React, { FunctionComponent, ReactNode } from "react";
 
 import styles from "./Header.css";
 
-type Props = PropTypesOf<typeof Typography>;
+interface Props {
+  children: ReactNode;
+  className?: string;
+  htmlFor?: string;
+  component?: "legend" | "label";
```

Comment:
This combination of props is unclear. Consider using `container?` with a react dom type instead where `htmlFor` can be injected on the passed in component wrapper.